src/app/authentication/authentication.service.ts

```diff
@@ -63,7 +63,8 @@ export class AuthenticationService {
 
     const paramLength = 'access_token='.length;
     const token = this.window.location.href.substring(tokenIndex + paramLength, this.window.location.href.indexOf('&'));
-    this.location.replaceState('');
+    const pathWithoutHash = this.location.path(false).split('#')[0];
```

Comment:
is this variable used anywhere?

Reply:
Yeah it's used in the next line :)

Reply:
strange, i dont recall seeing the next line when i looked at the PR